lab_2_tokenize_by_bpe/main.py

```diff
@@ -14,6 +15,22 @@ def prepare_word(
     :param end_of_word: a token that signifies the end of word
     :return: preprocessed word
     """
+    if not (
+        isinstance(raw_word, str)
+        and (isinstance(start_of_word, str) or start_of_word is None)
+        and (isinstance(end_of_word, str) or end_of_word is None)
+    ):
+        return None
+
+    tokenized_word = list(raw_word)
+    if start_of_word:
+        tokenized_word.insert(0, start_of_word)
+    if end_of_word:
+        tokenized_word.append(end_of_word)
+
+    tokenized_word_tuple = tuple(tokenized_word)
```

Comment:
do not create this variable - return as is

Reply:
done